tarantool/response.py

```diff
@@ -248,3 +251,34 @@ def __str__(self):
         return ''.join(output)
 
     __repr__ = __str__
+
+
+class ResponseExecute(Response):
+    @property
+    def lastrowid(self):
+        if self.body is None:
+            raise InterfaceError("Trying to access data, when there's no data")
+        info = self.body.get(IPROTO_SQL_INFO)
+
+        if info is None:
+            return None
+
+        lastrowids = info.get(IPROTO_SQL_INFO_AUTOINCREMENT_IDS)
+
+        return lastrowids[-1] if lastrowids else None
+
+    @property
+    def rowcount(self):
+        if self._body is None:
+            raise InterfaceError("Trying to access data, when there's no data")
+
+        info = self._body.get(IPROTO_SQL_INFO)
+
+        if info is None:
+            return -1
+
+        return info.get(IPROTO_SQL_INFO_ROW_COUNT, -1)
```

Comment:
We should consider the following points, IMHO:

* Handle an error response and return None.
* A successful response is guaranteed to have IPROTO_SQL_INFO.
* A successful response may contain or not contain IPROTO_SQL_INFO_ROW_COUNT. In the latter case we should return None.

I would implement it like so:

```python
if self._return_code != 0:
    return None
sql_info = self._body[IPROTO_SQL_INFO]
return sql_info.get(IPROTO_SQL_INFO_ROW_COUNT, None)
```

Do not I miss anything?

Reply:
Please, see https://github.com/tarantool/tarantool-python/pull/161#discussion_r480967190